Zc-specification/Zc.adoc

```diff
@@ -284,6 +284,27 @@ The Zcmt extension depends on the <<Zca>> and Zicsr extensions.
 
 |===
 
+[#Zc_formats]
+=== Zc instruction formats
+
+[%header,cols="2,3,2,1,1,1,1,1,1,1,1,1,1"]
+|=====================================================================
+| Format | instructions       | 15 10  | 9  | 8 | 7   | 6 | 5 | 4   | 3 | 2  | 1 | 0
+| CLBU   | c.lbu              | funct6 3+| rs1'    2+| uimm    3+| rd'    2+| op
+| CSB    | c.sb               | funct6 3+| rs1'    2+| uimm    3+| rs2'   2+| op
+| CLH    | c.lhu, c.lh        | funct6 3+| rd'/rs1'| funct1 | uimm 3+| rd'  2+| op
+| CSH    | c.sh               | funct6 3+| rd'/rs1'| funct1 | uimm 3+| rs2' 2+| op
+| CSZN   | c.[sz]ext.*, c.not | funct6 3+| rd'/rs1' 5+| funct5            2+| op
+| CMVSA  | cm.mvsa01 cm.mva01s| funct6 3+| r1s     2+| funct2 3+| r2s     2+| op

```

Comment:
What about calling them rs1`` and rs2`` instead? So they are clearly defined as a different format of source register? (I am not sure if its better than what you suggested, think it might be clearer that they are source registers but the double ` might not be as clear)